### PR TITLE
roadmap update including blog

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
Includes a short blog describing our updated roadmap process and deprecates old content on roadmap documentation page.

### CODE REVIEW

File: docs/content/en/blog/articles/roadmap-update/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Prioritization and roadmaps are often among the more challenging areas for any open source project to manage. On the K8ssandra project, we're making some improvements to how we manage our roadmap for new features. 
+
+# Legacy Roadmap
+If you've poked around the website a bit, you may have noticed the [roadmap]({{< relref "docs/roadmap">}}) page that we've been using to list a non-prioritized list of changes that have been proposed by project contributors. 
+
+As part of our post-1.0 release cleanup work, we've identified that a more flexible, robust approach was needed, and that it would be much easier to track in Git. The old roadmap page will be available for a short time but eventually phased out.
+ 
+# New Roadmap 
+The new roadmap is implemented as a [Git project](https://github.com/orgs/k8ssandra/projects/6) under the main K8ssandra GitHub org. The roadmap is broken out by quarter, with a lane for each of the next 4 quarters, and a lane for other items that, while important, are not yet prioritized.
```
Comment (bradfordcp) [inline docs/content/en/blog/articles/roadmap-update/index.md:19]:
```suggestion
The new roadmap is implemented as a [GitHub project](https://github.com/orgs/k8ssandra/projects/6) under the K8ssandra [organization](https://github.com/k8ssandra/). The roadmap is broken out by quarter, with a lane for each of the next 4 quarters, and a lane for other items that, while important, are not yet prioritized. Currently, each item in this list is a "note", which boils down to a single text field. Over the coming days, these will be transitioned into issues within the roadmap [repository](https://github.com/k8ssandra/roadmap) which allow for richer integration with related resources inside Github (milestones, labels, etc) and a cleaner UI. 
```

File: docs/content/en/blog/articles/roadmap-update/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+As part of our post-1.0 release cleanup work, we've identified that a more flexible, robust approach was needed, and that it would be much easier to track in Git. The old roadmap page will be available for a short time but eventually phased out.
+ 
+# New Roadmap 
+The new roadmap is implemented as a [Git project](https://github.com/orgs/k8ssandra/projects/6) under the main K8ssandra GitHub org. The roadmap is broken out by quarter, with a lane for each of the next 4 quarters, and a lane for other items that, while important, are not yet prioritized. 
+
+Chris Bradford shared a quick overview of this new roadmap and contents at the most recent meeting of the Cassandra Kubernetes Special Interest Group (SIG). You can see the video here:
+
+{{< youtube id="RXAdWLKw450" yt_start="2315" >}}
+
+Going forward, we hope to use this roadmap to manage our work. We'll convert items on the project board into issues as work is started on them.
```
Comment (bradfordcp) [inline docs/content/en/blog/articles/roadmap-update/index.md:25]:
```suggestion
Going forward, we hope to use this roadmap to manage our work and define epics that span repository boundaries.
```

File: docs/layouts/shortcodes/youtube.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="embed-responsive embed-responsive-16by9">
+    <iframe src="https://www.youtube.com/embed/{{ .Get "id" }}?enablejsapi=1{{ with .Get "color" }}{{ if eq . "white" }}&color=white{{ end }}{{ end }}{{ with .Get "autoplay" }}{{ if eq . "true" }}&autoplay=1{{ end }}{{ end }}{{ if isset .Params "yt_start" }}&start={{ .Get "yt_start" }}{{ end }}{{ if isset .Params "yt_end" }}&end={{ .Get "yt_end" }}{{ end }}{{ with .Get "modestbranding" }}{{ if eq . "true" }}&modestbranding=1{{ end }}{{ end }}"{{ if .Get "class" }} class="{{ .Get "class" }}"{{ end }} allowfullscreen></iframe>
+</div>
```
Comment (bradfordcp) [inline docs/layouts/shortcodes/youtube.html:3]: Newline at end of file

```suggestion
</div>

```

File: docs/content/en/docs/roadmap/_index.md [hunk 4]
```diff
@@ -7,9 +7,11 @@ description: K8ssandra roadmap ideas for community consideration.
 
 K8ssandra today is deployed as an entire stack. This open-source technology currently assumes your deployment uses the entire stack. Trading out certain components for others is not supported at this time. As part of the roadmap, one goal is to support a la carte composition of components.
 
-The following additional ideas are not yet in priority order:
+The roadmap is currently tracked in GitHub on a [project board](https://github.com/orgs/k8ssandra/projects/6).
 
-* Preconfigured alerts for metrics
+The following is a previous version of the roadmap with ideas in no particular order:
```
Comment (bradfordcp) [inline docs/content/en/docs/roadmap/_index.md:12]: Is there a reason we're keeping this page if everything is going to be on the roadmap board?

**Review by bradfordcp — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/roadmap/_index.md (hunk 4 above)
Comment (jeffreyscarpenter) [reply docs/content/en/docs/roadmap/_index.md:12]: I was thinking we'd leave the current page as a breadcrumb with forwarding address to new location for some short amount of time

**Review by bradfordcp — APPROVED** [review]

**Pull request merged**